templates/base/client/.rtag/Forms.tsx.hbs

```diff
@@ -292,7 +292,7 @@ export function Forms({ open, setOpen }: { open: boolean; setOpen: (v: boolean)
           >
             <Dialog.Overlay className="absolute inset-0 transition-opacity bg-gray-500 bg-opacity-75" />
           </Transition.Child>
-          <div className="fixed inset-y-0 right-0 flex max-w-full pl-10">
+          <div className="max-z-index fixed inset-y-0 right-0 flex max-w-full pl-10">
```

Comment:
this didn't end up working right?

Reply:
Nice catch removed this class + css rules